schemas/api_schema.js

```diff
@@ -174,6 +175,35 @@ type Mutation {
                     where: { category },
                     limit: count
                 });
+            },
+            SearchSessions: (parent, { keywords }, { db }, info) => {
```

Comment:
This will work but this is not scaleable at all, I'm fine with this for now - but please put a comment to remind us that this is not good enough